examples/ecs/entity_disabling.rs

```diff
@@ -0,0 +1,155 @@
+//! Disabling entities is a  powerful feature that allows you to hide entities from the ECS without deleting them.
+//!
+//! This can be useful for implementing features like "sleeping" objects that are offscreen
+//! or managing networked entities.
+//!
+//! Note that you should *not* use this feature to simply make entities invisible!
+//! [`Visibility`](bevy::prelude::Visibility) should be used for that:
+//! disabled entities are skipped entirely, which can lead to subtle bugs.
+//!
+//! # Default query filters
+//!
+//! Under the hood, Bevy uses a "default query filter" that skips entities with the
+//! the [`Disabled`] component.
+//! These filters act as a by-default exclusion list for all queries,
+//! and can be bypassed by explicitly including these components in your queries.
```

Comment:
Maybe provide an example of what "explicitly including components in the query" looks like.
`e.x 'Query<(A, B, Disabled)>' or 'Query<(A, B), With<Disabled>>'`